scripts/build/expected_all_platform_commands.txt

```diff
@@ -111,6 +111,12 @@ export ZEPHYR_TOOLCHAIN_VARIANT=cross-compile;
 export CROSS_COMPILE=$TELINK_TOOLCHAIN_PATH/riscv32-elf-;
 west build -d {out}/telink-tlsr9518adk80d-light -b tlsr9518adk80d {root}/examples/lighting-app/telink'
 
+# Generating tizen-tizen-light
```

Comment:
tizen-tizen souds odd ... it seems like this is arm (or arm64)? could we reuse that board name to have things read like `tizen-arm-ligth` ?

Reply:
I reuse board name for generating tizen-arm-light.